foursight_core/react/api/aws_stacks.py

```diff
@@ -41,7 +41,7 @@ def _create_aws_stack_info(stack: object):
 
 
 @memoize
-def aws_get_stack(stack_name_or_object: Union[str, object]) -> object:
```

Comment:
Ah, that is a good idea, in general, I didn't know about that, enumerated dict items. In this case I revisited the actual type since you brought it up, and was able to import the actual type (thought I'd had problems with this before), i.e.  ```from cryptography.hazmat.backends.openssl.rsa import _RSAPublicKey as RSAPublicKey```